src/TimeService.Data/WorkTimeMonthLimitRepository.cs

```diff
@@ -28,6 +38,53 @@ public Guid Add(DbWorkTimeMonthLimit workTimeMonthLimit)
             return workTimeMonthLimit.Id;
         }
 
+        public bool Edit(Guid workTimeMonthLimitId, JsonPatchDocument<DbWorkTimeMonthLimit> request)
+        {
+            DbWorkTimeMonthLimit dbWorkTimeMonthLimit = _provider.WorkTimeMonthLimits.FirstOrDefault(l => l.Id == workTimeMonthLimitId)
+                ?? throw new NotFoundException($"No worktime month limits with id {workTimeMonthLimitId}");
+
+            request.ApplyTo(dbWorkTimeMonthLimit);
+            dbWorkTimeMonthLimit.ModifiedAtUtc = DateTime.UtcNow;
+            dbWorkTimeMonthLimit.ModifiedBy = _httpContextAccessor.HttpContext.GetUserId();
```

Comment:
Why not write before ApplyTo method?

Reply:
and why  should it be before apply to method?

Reply:
this it would be logically correct, when read code

Reply:
why?, for me more correct is apply changes and after it update date changes, but it's still up to the save and if something happens, both will not be saved and the data will remain correct

Reply:
I agree with Egor, the provider can throw an exception